src/controllers/CommentController.js

```diff
@@ -0,0 +1,124 @@
+import { Comment, Article } from '../db/models';
+/**
+ * @description This controller handles comment request
+ * @class CommentController
+ */
+class CommentController {
+  /**
+   * @static
+   * @description The create comment method
+   * @param  {object} req The req object
+   * @param  {object} res The res object
+   * @returns {object} json res
+   * @memberof CommentController
+   */
+  static async addCommentToArticle(req, res) {
+    const article = res.locals.articleObject;
+    const articleId = article.id;
+    const { commentBody } = req.body;
+    const userId = req.user;
+
+    const {
+      userName,
+      bio,
+      imageUrl
+    } = article.User;
+
+    const commentObject = {
+      userId,
+      articleId,
+      commentBody
+    };
+
+    const comment = await Comment.create(commentObject);
+
+    const { id, createdAt, updatedAt } = comment;
+    try {
+      if (!comment) {
+        return res.status(500).json({
```

Comment:
I believe this should be a `404` error. Please confirm and make the change if necessary.